src/main/java/lottomanual/domain/Lotto.java

```diff
@@ -0,0 +1,83 @@
+package lottomanual.domain;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class Lotto {
+
+	public static final int NUMBERS_SIZE_LIMIT = 6;
+
+	private final Set<LottoNumber> numbers;
+
+	public Lotto(String numbersText) {
+		this(toSet(toList(numbersText)));
+	}
+
+	public Lotto(List<Integer> numbers) {
+		this(toSet(numbers));
+	}
+
+	private static List<Integer> toList(String text) {
+		String[] numberTexts = text.split(",");
+		validateSize(numberTexts.length);
+		return Arrays.stream(numberTexts).map(s -> Integer.parseInt(s.trim())).collect(Collectors.toList());
+	}
+
+	private static Set<LottoNumber> toSet(List<Integer> numbers) {
+		Set<LottoNumber> lottoNumbers = new TreeSet<>();
+		for (Integer number : numbers) {
+			lottoNumbers.add(LottoNumber.valueOf(number));
+		}
+		return lottoNumbers;
+	}
+
+	public Lotto(Set<LottoNumber> numbers) {
+		validateSize(numbers.size());
+		this.numbers = numbers;
+	}
```

Comment:
```suggestion
	private Lotto(Set<LottoNumber> numbers) {
		validateSize(numbers.size());
		this.numbers = numbers;
	}
	
	public static Lotto of(Set<LottoNumber> numbers) {
	        return new Lotto(numbers);
	}

	public static Lotto of(List<LottoNumber> numbers) {
		return of(new TreeSet<>(numbers));
	}
	
	public static Lotto of(String numbersText) {
		return Arrays.stream(text.split(","))
		        .map(LottoNumber::valueOf)
		        .collect(collectingAndThen(toList(), Lotto::of));
	}
```
이렇게 표현할 수 있을 것 같아요!
혹은 `.collect`에서 아예 TreeSet으로 변환하는 방법도 있답니다 ㅎㅎ

Reply:
정적 팩토리 메서드에 대해 한번 더 생각해볼 수 있었습니다!
피드백 감사합니다 ^^